uwblueprint/community-fridge-kw
# fix: dashboard empty state and donation details updates from visual enhancements

## Brief description. What is this change? 
### [Dashboard Flow Enhancements](https://www.notion.so/uwblueprintexecs/Team-Hub-Phase-1-Enhancements-Phase-2-cf7744a5b04748059761c1649a61a463?p=39b5b3c869274d01b9228bf98b32bcca)

Styled the empty donation state on dashboard home page and added updated styling to donation details. Also added Delete Donation section that deletes a scheduled donation (not hooked up with popup modal though).

## Implementation description. How did you make this change?
* [Desktop View](https://drive.google.com/file/d/1yDuGseH6Z6vhMdLm22d2fIZDalT7QfWW/view?usp=sharing)
* [Mobile View](https://drive.google.com/file/d/18DPeSazXr1AS4uvSfDGBDwBM4jWETWYK/view?usp=sharing)

## Steps to test
1. Schedule some donations, delete them, check empty state of dashboard page.
2. Check if it's close to Figma designs

## Checklist
